qiskit/providers/__init__.py

```diff
@@ -488,10 +488,10 @@ def has_realtime_logic(circuit: QuantumCircuit) -> bool:
 handles submitting the circuits to the backend to be executed and returning a
 :class:`~qiskit.providers.Job` object. Depending on the type of backend this
 typically involves serializing the circuit object into the API format used by a
-backend. For example, on IBM backends from the ``qiskit-ibm-provider``
-package this involves converting from a quantum circuit and options into a
-:mod:`.qpy` payload embedded in JSON and submitting that to the IBM Quantum
-API. Since every backend interface is different (and in the case of the local
+backend. For example, a provider service might need to convert from a quantum
+circuit and options into a
```

Comment:
Maybe simply restore the sentence for qiskit-ibm-runtime?

```suggestion
backend. For example, on backends from the ``qiskit-ibm-runtime`` 
package this involves converting from a quantum circuit and options into a
```

Reply:
I went for a very simplified version in 5f0c52c8c6d5625c473e1a6e7c9d5eddbbe15850 . Have a look.